src/LDLFactorizations.jl

```diff
@@ -2,18 +2,67 @@ module LDLFactorizations
 
 export ldl, \
 
-using AMD, SparseArrays
+using AMD, SparseArrays, LinearAlgebra
 
 mutable struct SQDException <: Exception
   msg::String
 end
 
-function ldl_symbolic!(n, Ap, Ai, Lp, parent, Lnz, flag, P, Pinv)
-  # compute inverse permutation
-  @inbounds for k = 1:n
-    Pinv[P[k]] = k
+"""
```

Comment:
Add a line here with the calling sequence for this function: `col_symb!(n, Ap, Ai, Cp, w, Pinv)` (like the function `bar` at the top of [this page](https://docs.julialang.org/en/v1/manual/documentation/))